client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-meeting/components/meeting-import/meeting-import.component.ts

```diff
@@ -47,7 +50,18 @@ export class MeetingImportComponent extends BaseComponent implements OnInit {
             const meeting = await new Promise<ImportMeeting>(resolve => {
                 const reader = new FileReader();
                 reader.addEventListener(`load`, progress => {
-                    const result = JSON.parse(progress.target.result as string);
+                    let result;
+                    try {
+                        result = JSON.parse(progress.target.result as string);
+                    } catch (e) {
+                        this.snackbar.open(
+                            `${this.translate.instant(`Error`)}: ${this.translate.instant(
+                                WRONG_JSON_IMPORT_FORMAT_ERROR_MSG
+                            )}`,
+                            `OK`
+                        );
+                        throw new Error(WRONG_JSON_IMPORT_FORMAT_ERROR_MSG);
+                    }
```

Comment:
Please move this whole function to a higher level class to deduplicate the code. As far as I see it, the only difference to the code in the workflow import is the used repository, so this should be easy to do.